src/60_Samples_%26_Templates/Hotel.html

```diff
@@ -17,279 +18,149 @@
 <script async src="https://cdn.ampproject.org/v0.js"></script>
 <!-- ## Setup -->
 <!--
-Import the additional AMP components. This sample uses fitted text, an image carousel, a form, mustache templates, an iframe, and a list.
+  Import the additional AMP components. We use `amp-form` to send a filter request to the server.
```

Comment:
We use a few AMP extensions to dynamically render a list of rooms.

Reply:
done